packages/compass-components/src/hooks/use-stacked-component.tsx

```diff
@@ -75,19 +62,3 @@ const withBaseStyles = function <UsePopover extends boolean, ComponentProps>(
     ComponentWithStackedStyles
   ) as typeof WrappedComponent;
 };
-
-export const withStackedComponentStyles = function <
-  ComponentProps extends { className?: string }
->(
-  WrappedComponent: ComponentType<ComponentProps>
-): ComponentType<ComponentProps> {
-  return withBaseStyles(WrappedComponent, false);
-};
-
-export const withStackedComponentPopoverStyles = function <
-  ComponentProps extends { popoverZIndex?: number }
->(
-  WrappedComponent: ComponentType<ComponentProps>
-): ComponentType<ComponentProps> {
-  return withBaseStyles(WrappedComponent, true);
-};
```

Comment:
this becomes irrelevant with packages bump and `Popover` has marked `popoverZIndex` as deprecated.

Reply:
Is there another way to provide the index now? We need to confirm this doesn't break compass-web in mms

Reply:
Yes, i'll check this with mms integration (using sync-with-mms script)